docs/_documentations/overview.md

```diff
@@ -9,21 +9,72 @@ type: document
 parent: root
 ---
 
-# Overview
+# What is Codewind? - Product Overview
+Codewind is an [open source project](https://github.com/eclipse/codewind) for building and developing cloud-native, containerised applications. Built from the ground up, Codewind provides an improved and integrated IDE-based container development flow. We're all about the developer experience. 
 
-You can use codewind to develop and debug your containerized projects from within an IDE.
+From creating your project, all the way to debugging and optimising, Codewind does the heavy-lifting so that you can simply focus on rapidly iterating your code. Expose the metrics and logs that you need and perform load testing at the click of a button, and relax in a flow that will feel the same as a traditional application flow.
 
-Write code, track application and build statuses, view project logs, and run your application. Codewind supports development of Appsody, Go, Java Lagom, Microprofile/Java EE, Node.js, Python, Spring, and Swift containerized projects. In addition, the tools support easily debugging Microprofile/Java EE, Spring, and Node.js applications.
+When it comes to running your microservices on the cloud, Codewind enables you to build and run Docker-containerised applications on [Kubernetes](https://kubernetes.io/) container orchestration applications, including [Openshift](https://www.openshift.com/), without the need for in-depth knowledge of container technology.
 
+## How does Codewind help me develop containerised applications?
+Codewind improves your inner loop experience, enabling you to create a microservice quickly, rapidly iterate on changes and make improvements to performance thanks to the following features:
 
-# Features
+1. **Access to pre-configured containerised project templates** Codewind provides pre-configured, Docker-containerised project templates ADD URL covering several languages including [Node.js](https://nodejs.dev/), Java, Python and [Swift](https://swift.org/), and several frameworks such as Express, Spring Boot and Open Liberty ADD URLS, with more on the way. [Quickly create and deploy microservices in languages and frameworks that you're familiar with](https://www.youtube.com/watch?v=zKMggp10gq4&t=12s). Modify the pre-configured projects to develop your customised microservice.
+
+2. **Ability to add your own applications** Access the seamless container development and deployment experience on your IDE that Codewind offers for your current projects.
+
+3. **Auto-build-and-run on code changes** As you develop, Codewind *automatically* pushes your code changes to your container as efficiently as possible. 
```

Comment:
- I think we've been avoiding italics...might need to get clarification with the team on this...

Reply:
Italics: raising as part of review and discussion of our Codewind style with Andy & Rob.

Reply:
Italics taken out for now.